src/index.css

```diff
@@ -61,4 +61,8 @@ code {
 /* Ensure smooth scrolling for the entire toolbox content */
 .blocklyToolboxContents {
   scroll-behavior: smooth !important;
-}
\ No newline at end of file
+}
+
+.eventHandlerButtonStyle {
+    fill: #995ba5;
```

Comment:
Is it possible (it might not be) to use the style in styles so if a different theme is selected it will still match?

Reply:
Done.